Dayanand Viveki/HTML/Assignment6/personaldetails.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="assets/css/style.css" />
+        <title>Personal Details</title>
+    </head>
+    <body>
+        <div class="banner">
+            <div class="headline font-size-35">
+                <p>MI-11 Inter HTML Assesment</p>
+            </div>
+            <hr />
+            <div class="headline font-size-25 italic-font">
+                <p>Personal Details</p>
+            </div>
+            <div class="form-item">
+                <form action="">
+                    <span>
+                        <label for="fname">Frist Name:</label>
+                        <input type="text" id="fname" placeholder="First Name" />
```

Comment:
placeholders second name should start with small letter like First name make this change in all input boxes

Reply:
ok mdm